deluge_web_client/client.py

```diff
@@ -23,9 +25,8 @@ def __enter__(self) -> "DelugeWebClient":
         self.login()
         return self
 
-    def __exit__(self, exc_type, exc_value, traceback) -> None:
-        """End of with statement, closes the session."""
-        self.close_session()
+    def __exit__(self, exc_type, exc_value, traceback) -> bool:
+        """End of with statement."""
         return False
```

Comment:
Thanks for this! We could actually remove the return statement and just `-> None` tbh. I don't think it was necessary to return `False`. 

Also if we re-add `close_session()` I think it makes sense to be put back here right?

Reply:
Yes, I don't see what the return value actually provides, and given that its always False, you could just `return` - yielding NoneType.

Reply:
Sounds good, if you would like to make that change as well!